pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -19,8 +19,8 @@ in
 with haskellLib;
 
 self: super: {
-  # enable list-transformer, jailbreaking is necessary until next release >0.13.0: https://github.com/ivanperez-keera/dunai/issues/427
```

Comment:
This was released, but it made the flag `transformers` conditionally increase the `transformers` upper-bound, so only the jailbreak can be removed. 
Very unusual situation, so I'm tempted to keep linking to the upstream issue even though it's closed.